workers/unity/Assets/Gdk/Core/Tests/Editmode/GameObjectRepresentation/GameObjectDispatcherCallbacksSystemTests.cs

```diff
@@ -0,0 +1,60 @@
+using Improbable.Gdk.TestUtils;
+using NUnit.Framework;
+using UnityEngine;
+
+namespace Improbable.Gdk.Core.EditmodeTests
+{
+    [TestFixture]
+    public class GameObjectDispatcherCallbacksSystemTests
+    {
+        private UnityTestWorker worker;
+
+        [SetUp]
+        public void Setup()
+        {
+            worker = new UnityTestWorker("worker-id", new Vector3());
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            worker.Dispose();
+        }
+
+        [Test]
+        public void OnCreateManager_initializes_GameObjectTranslation_ComponentGroups()
+        {
+            var world = worker.World;
+
+            foreach (var gameObjectTranslation in worker.View.GameObjectTranslations)
+            {
+                Assert.IsNull(gameObjectTranslation.ComponentAddedComponentGroup);
```

Comment:
This looks like it's just testing that a default constructed reference is null.

Reply:
removed those checks.